sw/device/lib/testing/sram_ctrl_testutils.h

```diff
@@ -23,32 +23,33 @@ typedef uint32_t (*sram_ctrl_testutils_exec_function_t)(volatile uint32_t,
 /**
  * A typed representation of the test data.
  */
-#define SRAM_CTRL_DATA_NUM_WORDS 4
-#define SRAM_CTRL_DATA_NUM_BYTES 128
-typedef struct sram_ctrl_data {
-  uint32_t words[SRAM_CTRL_DATA_NUM_WORDS];
-} sram_ctrl_data_t;
+#define SRAM_CTRL_TESTUTILS_DATA_NUM_WORDS 4
```

Comment:
These should not be below the comment for the struct itself, since that will cause it to attach in the wrong place.